lib/aurora_bootstrapper/notifier.rb

```diff
@@ -12,7 +14,27 @@ def export_date
     
     # ENV is string to string dictionary
     def export_date_override
-      DateTime.now.strftime("%Y-%m-%d") if ENV.key?('EXPORT_DATE_OVERRIDE')
+      return nil unless ENV.key?('EXPORT_DATE_OVERRIDE')
+
+      today = Date.today
+      days_ago = (0..30).find do | days_ago |
+        exists_export? date: today-days_ago
+      end
+      
+      converted_days_ago = (days_ago.nil? || days_ago.to_i == 0) ? 1 : days_ago.to_i
```

Comment:
super duper nit. Can we move the `to_i` into the line above and consolidate all the math into `days_ago` definition, to make this something like

```ruby
  today = Date.today
  days_ago = (0..30).find do | days_ago |
    exists_export? date: today-days_ago
  end.to_i

  days_ago -= 1 unless days_ago.zero?
  ( today - days_ago ).strftime( DATE_FORMAT )
```

Reply:
Cool Ruby chaining